draft-ietf-privacypass-auth-scheme.md

```diff
@@ -187,12 +141,13 @@ values MUST be ignored.
 hostname that is used to identify the issuer that is allowed to issue
 tokens that can be redeemed by this origin.
 
-- "redemption_nonce" is an optional field. If present, it indicates that
-a client needs to present an interactive token, generated specifically in
-response to this challenge. If empty, the client can use a non-interactive
-token. When present, this valid is a fresh 32-byte nonce generated by the
-origin for each challenge. Valid lengths for this field are either 0 or 32
-bytes. Challenges with redemption_nonce values of invalid lengths MUST
+- "redemption_context" is an optional field. If present, it allows the origin
+to require that clients fetch tokens bound to a specific context, as opposed
+to reusing tokens that were fetched for other contexts. When present, this valid
+is a fresh 32-byte context generated by the origin. This value can be a unique
```

Comment:
```suggestion
is a 32-byte context generated by the origin. This value can be a unique
```

It doesn't need to be "fresh" since one can compute it deterministically.